test/TestSites/CreateDefaultBuilderApp/Program.cs

```diff
@@ -36,6 +36,11 @@ static void Main(string[] args)
                     });
                 })
                 .Build().Run();
+
+            Console.ReadKey();
```

Comment:
This doesn't add any coverage for the new overload other than for compilation. Create a copy of this app that uses `CreateDefaultBuilder<Startup>` and add an test like https://github.com/aspnet/MetaPackages/blob/dev/test/Microsoft.AspNetCore.FunctionalTests/WebHostFunctionalTests.cs#L51 instead.